transient/transient.py

```diff
@@ -41,11 +41,19 @@ def __qemu_added_devices(self) -> List[str]:
             if self.config.ssh_console is True:
                 new_args.append("-nographic")
 
-            # Use userspace networking (so no root is needed), and bind
             # the random localhost port to guest port 22
-            self.ssh_port = self.__allocate_random_port()
-            new_args.extend(["-net", "nic,model=e1000",
-                             "-net", "user,hostfwd=tcp::{}-:22".format(self.ssh_port)])
+            if not self.config.ssh_port:
```

Comment:
I would use `if self.config.ssh_port is None` here.

Reply:
Good call